README.md

```diff
@@ -0,0 +1,274 @@
+
+2.  Configure the CLI, changing the values below as appropriate for your
+    local installation of DC/OS:
+
+        dcos cluster setup http://dcos-ea-1234.us-west-2.elb.amazonaws.com
+
+3.  Get started by calling the DC/OS CLI help:
+
+        dcos help
+
+## Running tests
+
+### Setup
+
+Before you can run the DC/OS CLI integration tests, you need to get a
+cluster up and running to test against. Currently, the test suite only
+supports testing against Enterprise DC/OS.
+
+Given these constraints, the easiest way to launch a cluster with these
+capabilities is to use
+[dcos-launch](https://github.com/dcos/dcos-launch) with the
+configuration listed below:
+
+    launch_config_version: 1
+    deployment_name: ${CLI_TEST_DEPLOYMENT_NAME}
+    installer_url: ${CLI_TEST_INSTALLER_URL}
+    platform: aws
+    provider: onprem
+    aws_region: us-west-2
+    aws_key_name: ${CLI_TEST_SSH_KEY_NAME}
+    ssh_private_key_filename: ${CLI_TEST_SSH_KEY_PATH}
+    os_name: cent-os-7
+    instance_type: m4.large
+    num_masters: 1
+    num_private_agents: 1
+    num_public_agents: 1
+    dcos_config:
+        cluster_name: DC/OS CLI Integration Tests
+        resolvers:
+            - 10.10.0.2
+        dns_search: us-west-2.compute.internal
+        master_discovery: static
+
+Where `CLI_TEST_DEPLOYMENT_NAME` is a custom name set by the user,
+`CLI_TEST_INSTALLER_URL` is the URL of a dcos_generate_config.ee.sh
+script for the desired version of DC/OS to test against,
+`CLI_TEST_SSH_KEY_NAME` is the name of an AWS key to install on the
+machines deployed by the installer, and `CLI_TEST_SSH_KEY_PATH` is a
+local path to the key named by `CLI_TEST_SSH_KEY_NAME`.
+
+Unfortunately, the URL to download `dcos_generate_config.ee.sh` scripts
+for Enterprise DC/OS is not publicly available. For Mesosphere employees
+the URL to the latest master build of Enterprise DC/OS can be found
+here:
+
+    https://mesosphere.onelogin.com/notes/45791
+
+For everyone else, you can still run the integration test suite against
+a non-enterprise cluster (i.e. Community DC/OS), but please be aware
+that running the full test suite *will* fail. See the section below on
+*Running* to see how to limit the set of tests run by the integration
+test suite.
+
+The URL to the latest master build of Community DC/OS is:
+
+    https://downloads.dcos.io/dcos/testing/master/dcos_generate_config.sh
+
+### Initialization
+
+Once you have your cluster up and running you need to modify your
+environment in order to run the tests. A simple script you can use to
+modify your environment can be seen below.
+
+*NOTE*: Make sure you run this script from your **top-level** dcos-cli
+directory (i.e. **not** inside dcos-cli/cli).
+
+*NOTE*: You will need to customize the first few lines in the script
+appropriate for your setup. A description of the variables you need to
+modify can be found below the script
+
+*NOTE*: The script will modify your **global** `/etc/hosts` file. This is
```

Comment:
I forgot to remove this note in https://github.com/dcos/dcos-cli/commit/58bab4b1fb565cd96c7ba41f71851fc297a7ef82, but this is not the case anymore.